### PR TITLE
Add orders details and let the user configure the order status after payment

### PR DESCRIPTION
**Pull request by msaustral**
Se agregan campos buy order, token, responseCode

Se resuelve issue https://github.com/TransbankDevelopers/transbank-plugin-prestashop-webpay-rest/issues/7

Se coloca en panel izquierdo de la orden ocupando solo 6-col

### CODE REVIEW

**Comment by msaustral** [comment]
Se agrega la opción para cambiar el estatus que tendrán las órdenes al ser aceptado el pago en la configuración del módulo.

esto cierra los casos: 
https://github.com/TransbankDevelopers/transbank-plugin-prestashop-webpay-rest/issues/8
https://github.com/TransbankDevelopers/transbank-plugin-prestashop-webpay-rest/issues/10

**Review by msaustral — COMMENTED** [review]
Se agrega la opción para cambiar el estatus que tendrán las ordenes al ser aceptado el pago en la configuración del módulo.

esto cierra los casos: #8

#10

File: webpay/webpay.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function __construct() {
         
         $this->name = 'webpay';
         $this->tab = 'payments_gateways';
-        $this->version = '1.0.2';
+        $this->version = '1.0.3';
```
Comment (gonzunigad) [inline webpay/webpay.php:39]: This will change in the next PR.

Comment (gonzunigad) [reply webpay/webpay.php:39]:
```suggestion
        $this->version = '1.0.2';
```

File: webpay/config.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <module>
 	<name>webpay</name>
 	<displayName><![CDATA[Webpay Plus]]></displayName>
-	<version><![CDATA[1.0.0]]></version>
+	<version><![CDATA[1.0.1]]></version>
```
Comment (gonzunigad) [inline webpay/config.xml:5]:
```suggestion
	<version><![CDATA[1.0.0]]></version>
```
This should always be 1.0.0 so the deployment can put the right value in here.

File: webpay/config_es.xml [hunk 3]
```diff
@@ -2,11 +2,12 @@
 <module>
     <name>webpay</name>
     <displayName><![CDATA[Webpay Plus]]></displayName>
-    <version><![CDATA[1.0.0]]></version>
-    <description><![CDATA[Recibe pagos en linea con Tarjetas de crédito y Redcompra en tu Prestashop a través de Webpay Plus REST]]></description>
+    <version><![CDATA[1.0.2]]></version>
```
Comment (gonzunigad) [inline webpay/config_es.xml:5]:
```suggestion
    <version><![CDATA[1.0.0]]></version>
```
This should always be 1.0.0 to the deployment script to work.

File: webpay/controllers/front/validate.php [hunk 4]
```diff
@@ -163,7 +163,7 @@ private function processPayment($webpayTransaction, $cart)
             $currency = Context::getContext()->currency;
             $OKStatus = Configuration::get('WEBPAY_DEFAULT_ORDER_STATE_ID_AFTER_PAYMENT');
             if ($OKStatus === "0") {
-                $OKStatus = Configuration::get('PS_OS_PREPARATION');
+                $OKStatus = Configuration::get('PS_OS_PAYMENT');
```
Comment (gonzunigad) [inline webpay/controllers/front/validate.php:166]:
```suggestion
                $OKStatus = Configuration::get('PS_OS_PREPARATION');
```
This change could make the update not backwards compatible.

File: webpay/views/templates/admin/config.tpl [hunk 5]
```diff
@@ -27,6 +27,15 @@
...
 		<div class="margin-form"><input type="text" size="90" id="apiKeySecret" name="apiKeySecret" value="{$data_apikeysecret|escape:'htmlall':'UTF-8'}"/></div>
+		
+		<br/>
+		<label for="webpay_default_order_state_id_after_payment">{l s='Estado pago aceptado' mod='webpay'}</label>
+		<div class="margin-form">
+			<select name="webpay_default_order_state_id_after_payment">
+				<option value="{$pago_aceptado_id}" {if $default_after_payment_order_state_id eq $pago_aceptado_id}selected{/if}>Pago aceptado</option>
+				<option value="{$preparacion_curso_id}" {if $default_after_payment_order_state_id eq $preparacion_curso_id}selected{/if}>Preparación en curso</option>
+			</select>
+		</div>
```
Comment (gonzunigad) [inline webpay/views/templates/admin/config.tpl:38]:
```suggestion
		
		<br/>
		<label for="webpay_default_order_state_id_after_payment">{l s='Estado pago aceptado' mod='webpay'}</label>
		<div class="margin-form">
			<select name="webpay_default_order_state_id_after_payment">
				<option value="{$pago_aceptado_id}" {if $default_after_payment_order_state_id eq $pago_aceptado_id}selected{/if}>Pago aceptado</option>
				<option value="{$preparacion_curso_id}" {if $default_after_payment_order_state_id eq $preparacion_curso_id}selected{/if}>Preparación en curso</option>
			</select>
		</div>
```


Looks great, thanks. Is is possible to use English for the variable names?
$preparationStatusId and $paymentAcceptedStatusId should work :)

File: webpay/config_es.xml (hunk 3 above)
Comment (gonzunigad) [reply webpay/config_es.xml:5]: I think this is should also be 1.0.0

**Review by gonzunigad — APPROVED** [review]

File: webpay/config_es.xml (hunk 3 above)
Comment (msaustral) [reply webpay/config_es.xml:5]: done

Comment (gonzunigad) [inline webpay/config_es.xml:5]:
```suggestion
    <version><![CDATA[1.0.0]]></version>
```

**Review by gonzunigad — APPROVED** [review]
Thanks for the great work!

**Pull request merged**